app/src/main/java/org/wikipedia/suggestededits/SuggestedEditsTasksFragment.kt

```diff
@@ -187,22 +188,28 @@ class SuggestedEditsTasksFragment : Fragment() {
 
                     totalContributions += wikidataResponse.query?.userInfo()!!.editCount
                     totalContributions += commonsResponse.query?.userInfo()!!.editCount
+                    totalContributions += localDescriptionsContributions.size
```

Comment:
But couldn't we just say:
`totalContributions += homeSiteResponse.query?.userInfo()!!.editCount`

(and change the homeSite call to use `getUserContributions` instead of `userInfo`)
And wouldn't that automatically take care of https://phabricator.wikimedia.org/T287031 ?

Reply:
It makes sense to me. But I am also thinking that we should still keep the `localDescriptionsContributions` because we will need the article titles to get pagevIews in the past 30 days.